apps/core/mutations.py

```diff
@@ -54,10 +55,10 @@ def mutate(root, info, data):
         return CreateDataset(result=dataset, errors=None, ok=True)
 
 
-class UpdateTable(DiveMutationMixin):
+class AddTableToWorkSpace(DiveMutationMixin):
     class Arguments:
         data = TableInputType(required=True)
```

Comment:
Should this not contain just `isAddedToWorkspace`? `TableInputType` will allow for other items to be changed as well. I don't think we want to allow other modifications in this API.

Reply:
thanks